crates/ide_completion/src/tests/record.rs

```diff
@@ -129,9 +99,54 @@ fn foo(f: Struct) {
 #[test]
 fn functional_update() {
     // FIXME: This should filter out all completions that do not have the type `Foo`
```

Comment:
I don't know if that's really doable btw, since you might write an expression evaluating to `Foo`...

Reply:
Ye filter is probably the wrong idea here, prioritizing should be the goal.